posts/rails_memory.md

```diff
@@ -3,14 +3,20 @@ title: "Rails memory leak problems"
 date: "2021-10-10"
 ---
 
+<!-- Checking.... -->
+
 Ruby memory leak problems are maybe familiar with Rubyist and you can find so many articles about them palso the solution with detailed explanation. As a Rails developer, I also stuck on that, and I want to share my experience. Let's start!
 
 # Problem showed up
 
+<!-- part 1 -->
+
 One morning, I woke up and received a message from the analysis team. They received 502 code when calling APIs. I started to investigate, of course, from server log. I went to Cloudwatch Log and found something were abnormal.
 
 <img src="/images/high-memory-graph.png" />
 
+<!-- part 1.5 -->
+
 When analysis team called APIs, the memory percentage raise high, it is normal. But after that, the memory percentage did not decrease. And after 2-3 days, the memory went to 99%. That were unacceptable. And if I restart the server, it becames normal for the first day, after that, it's memory went up crazyly.
 
 At first, I thought maybe it were thread or process or gem memory leak problems. So I started to check.
```

Comment:
- It were -> It was
- I thought maybe -> I assumed

Reply:
https://github.com/HoangNguyen679/aphelios/pull/1/commits/5304e5be3281d5c2a0e438f5efdab526f961704e fixed